### PR TITLE
Update memory estimates table

### PR DESCRIPTION
**Pull request by archigup**
Updates the memory estimates doxygen table to be in line with numbers on Freertos.org

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by leegeth — APPROVED** [review]

**Comment by archigup** [comment]
Capture

**Comment by archigup** [comment]
Capture

File: docs/doxygen/pages.dox [hunk 1]
```diff
@@ -17,19 +17,17 @@ Embedded devices often use MQTT to interact with Device Shadow. This library pro
...
+        <td><center><b>With -Os Optimization</b></center></td>
     </tr>
     <tr>
         <td>shadow.c</td>
-        <td>2.3K</td>
-        <td>1.8K</td>
-        <td>0.9K</td>
+        <td><center>1.2K</center></td>
+        <td><center>0.7K</center></td>
     </tr>
```
Comment (leegeth) [inline docs/doxygen/pages.dox:31]: Should we have one more raw with total in order to be consistent with the table in FRO?

**Comment by archigup** [comment]
Capture

**Review by leegeth — APPROVED** [review]

**Review by cobusve — APPROVED** [review]

**Pull request merged**